.github/workflows/publish.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         fetch-depth: '1'
     - name: Bump version and push tag
       id: bump_tag
-      uses: anothrNick/github-tag-action@1.36.0
+      uses: anothrNick/github-tag-action@1.37.0
```

Comment:
i think it is now a 1.38.0 but fine like this as well
https://github.com/anothrNick/github-tag-action/releases/tag/1.38.0